config/environments/test.rb

```diff
@@ -9,10 +9,14 @@
 
     # These are hard to remove as they are in the admin controller who doesn't know
     # how to eager load the original case just for ICO cases
-    [Case::OverturnedICO::FOI, Case::ICO::FOI, Case::ICO::SAR, Case::OverturnedICO::SAR].each do |klass|
+    [Case::OverturnedICO::FOI, Case::ICO::FOI, Case::ICO::SAR, Case::OverturnedICO::SAR, Case::SAR::OffenderComplaint].each do |klass|
       Bullet.add_whitelist :type => :n_plus_one_query,
                            :class_name => klass.name,
                            :association => :original_case
+      # Basically the same thing as above, but one link further down
```

Comment:
What do you mean by "Above" here, the other `Bullet.add_whitelist` call?

Reply:
What I meant is the orignal_case association defined in linkable_original_case.rb 
`    has_one :original_case_link,
            -> { original },
            class_name: 'LinkedCase',
            foreign_key: :case_id

    has_one :original_case,
            through: :original_case_link,
            source: :linked_case`

is through the 'orignal_case_link' -> linked_cases table,  so those definitions seem to create a chain of associations to me and the original_case_link is further down on this chain :).   
I think I just remove this comment as it doesn't provide useful information.